pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!-- Title format: PAS-XXX | short pr description -->
+
+[Jira Ticket PAS-XXX](https://bitwarden.atlassian.net/browse/PAS-XXX)
+
+### Description
+<!-- Introduction that should allow the reviewer to quickly be able to understand the reason for opening this PR. -->
+
+<!-- If this is a shape (i.e. architectural spec) uncomment the section below: -->
+<!-- ### Shape
+Give a brief overview of your approach, the pros/cons that were considered, etc.
+
+ -->
+
+### Screenshots
+<!-- Include any screenshots if this is a UI change. Delete if N/A. -->
+
+I did the following to ensure that my changes were tested thoroughly to ensure safety:
+[] __
+[] __
```

Comment:
```suggestion
### Ticket

<!--
    Link the corresponding GitHub Issue(s) and/or Jira Task(s)
    that are addressed by this pull request.
-->

<!-- For GitHub Issues: -->
- Closes #xxx

<!-- For Jira Tasks: -->
<!-- (remove if external contributor) -->
- Closes [PAS-XXX](https://bitwarden.atlassian.net/browse/PAS-XXX)

### Description

<!--
    Introduction that should allow the reviewer to quickly be able to understand the reason for opening this PR.
-->

### Shape

<!--
    Give the high-level overview of the technical design involved in the implemented changes.
    If the changes don't have any architectural impact, you can remove this section.
-->

### Screenshots

<!--
    Include any relevant UI screenshots showcasing the before & after of your changes.
    If the changes don't have an UI impact, you can remove this section.
-->

### Checklist

I did the following to ensure that my changes were tested thoroughly:
- [ ] __
- [ ] __

I did the following to ensure that my changes did not introduce new security vulnerabilities:
- [ ] __
- [ ] __
```

Reply:
I don't know if multiline HTML comments work, but thought it looked nicer

Reply:
they work - thank you

Reply:
@Tyrrrz i updated the template with these changes!